src/app.jsx

```diff
@@ -188,7 +188,7 @@ export const DropdownBasic = () => {
             isOpen={isOpen}
             onSelect={onSelect}
             onOpenChange={isOpen => setIsOpen(isOpen)}
-            toggle={toggleRef => <MenuToggle ref={toggleRef} onClick={onToggleClick} isExpanded={isOpen} variant="secondary" splitButtonOptions={{ variant: "action", items: [<Icon key="list-icon"><ListIcon /></Icon>] }} />} ouiaId="BasicDropdown"
```

Comment:
Remove this last commit; this is the wrong way to add spacing in to this component. Let's fix it in followup.